packages/eslint-config-office-addins/package.json

```diff
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "@types/node": "^12.12.34",
+    "@types/es6-promise": "^3.3.0",
```

Comment:
Is this related to switching lerna versions? Seems like this PR is doing more than one thing.

Reply:
check your teams :-).  For some reason updating lerna versions exposed these missing pieces of a few package.json files causing the builds to break (somehow missed this when I build locally the first time).  That why the CI loop didn't pass the first time.